3_javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,117 @@
+//Age-ify (A future age calculator)
+
+let yearOfBirth = 1999;
+let yearFuture = 2027;
+
+let age = yearFuture - yearOfBirth;
+console.log(`you will be in ${age} years old in ${yearFuture}`);
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth = 2017;
+let dogYearFuture = 2027;
+let shouldShowResultInDogYears = true;
+ 
+
+let dogAgeInHumanYears = dogYearFuture - dogYearOfBirth;
+
+let dogAgeInDogYears = dogAgeInHumanYears * 7;
+
+if (shouldShowResultInDogYears) {
+  console.log(`your dog will be ${dogAgeInDogYears} dog years old in ${dogYearFuture}`);
+} else {
+  console.log(`your dog will be ${dogAgeInHumanYears} human years old in ${dogYearFuture}`);
+}
+
+
+
+
+//Housey pricey (A house price estimator)
+
+
+const friendsHouse = [
+  {
+    name: "Peter",
+    housePrice: 2500000,
+    width: 8,
+    depth: 10,
+    height: 10,
+    gardenSizeInM2: 100,
+  },
+  {
+    name: "Julia",
+    housePrice: 1000000,
+    width: 5,
+    depth: 11,
+    height: 8,
+    gardenSizeInM2: 70,
+  },
+];
+
```

Comment:
🏡 Appreciate the structured approach with the array of objects. This definitely adds scalability.